docs/DeveloperGuide.md

```diff
@@ -313,36 +313,45 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **MSS**
 
-1.  Pseudo steps
-2.  User requests to delete a specific assignment in the list.
-3.  Academic Schedule Manager deletes the assignment.
+1.  User retrieves NUSMods timetable URL from the NUSMods website.
+2.  User requests to import NUSMods timetable using their NUSMods timetable URL.
+3.  Academic Schedule Manager retrieves lesson data from NUSMods API.
+4.  Academic Schedule Manager adds all the lessons according to the data retrieved.
+5.  Academic Schedule Manager shows a success message with details of the lessons added.
```

Comment:
Looks good :) A minor nitpick is that you may want to consider removing point number 3 ("ASM retrieves lesson data from NUSMods API") as according to the textbook, use cases only describe the **externally visible behavior, not internal details, of a system**.

Reply:
Have edited accordingly. Thanks.